### PR TITLE
Monetization (MON-323): MonetizationModule - updated styles for amazon video and ecommerce

### PR DESCRIPTION
**Pull request by saipetch**
Monetization (MON-323): update styles for MonetizationModule

### CODE REVIEW

File: extensions/wikia/MonetizationModule/styles/MonetizationModule.scss [hunk 1]
```diff
@@ -422,3 +431,7 @@
 		padding: 10px 0 11px;
 	}
 }
+
+#monetization-amazon_video-above_title {
```
Comment (lizlux) [inline extensions/wikia/MonetizationModule/styles/MonetizationModule.scss]: Is there a reason why this class name doesn't match our coding conventions? https://github.com/Wikia/guidelines/blob/master/SCSS/CodingConventions.md#class-and-id-naming-conventions

File: extensions/wikia/MonetizationModule/styles/MonetizationModule.scss [hunk 2]
```diff
@@ -28,6 +28,10 @@
 		margin-bottom: 25px;
 		margin-top: 25px;
 
+		&.above_title {
```
Comment (lizlux) [inline extensions/wikia/MonetizationModule/styles/MonetizationModule.scss]: Ideally this would be `.above-title`.

**Comment by lizlux** [comment]
One coding convention comment, otherwise, LGTM.

**Comment by tedgill** [comment]
:+1:

**Pull request merged**